src/components/language_picker.html

```diff
@@ -0,0 +1,39 @@
+{% import "Macros" as macros %}
+<button id="language-toggle{{ counter }}" class="icon-button" type="button"
+    title="Change language" aria-label="Change language"
+    aria-haspopup="true" aria-expanded="false"
+    aria-controls="language-list{{ counter }}" style="{{ attributes | get(key="style", default="")  }}">
+    <i class="fa fa-globe"></i>
+</button>
+<ul id="language-list{{ counter }}" class="theme-popup" aria-label="Languages"
+    role="menu" style="left: auto; right: 10px;">
+    {% for identifier, language_name in get_context(key="output.i18n.languages") %}
+    <li role="none">
+        <a id="{{ identifier }}"
+            href="{{ macros::get_rendered_path(identifier=identifier) }}"
+            style="color: inherit;">
+            <button role="menuitem" class="theme {% if identifier == language %} theme-selected {% endif %}">
+                {% include "Included" %} {{ language_name }}
+            </button>
+        </a>
+    </li>
+    {% endfor %}
+</ul>
+
+<script>
+    let langToggle = document.getElementById("language-toggle{{ counter }}");
+    let langList = document.getElementById("language-list{{ counter }}");
+    {% raw %}
```

Comment:
We need a raw block for the Javascript block

Reply:
Okay, but then I think the `{{` below has a `{` too many?

Reply:
Yes, that's a mistake actually (from copy pasting the HBS one), but still valid JS